slack.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: redirect
+title: Hack and HHVM Slack
+id: slack
+destination: https://forms.gle/io5oYV1CunwZUC2EA
```

Comment:
This is what matters, but keeping the content below just in case the redirect fails. Works locally